src/pages/basic_data/app_admin/central_import/DuplicateSearchFilterForm.tsx

```diff
@@ -0,0 +1,270 @@
+  publication,
+  setDuplicateSearchFilters,
+}: DuplicateSearchFilterFormProps) => {
+  const { t } = useTranslation('basicData');
+
+  const initialSearchParams = {
+    doi: publication.entityDescription?.reference?.doi ?? '',
+    title: publication.entityDescription?.mainTitle ?? '',
+    author: publication.entityDescription?.contributors[0]?.identity.name ?? '',
+    issn: '',
+    yearPublished: '',
+    isDoiChecked: true,
+    isTitleChecked: false,
+    isAuthorChecked: false,
+    isIssnChecked: false,
+    isYearPublishedChecked: false,
+  };
+
+  return (
+    <Formik
+      initialValues={initialSearchParams}
+      onSubmit={(values) => {
+        setDuplicateSearchFilters({
+          doi: values.isDoiChecked ? values.doi : '',
+          title: values.isTitleChecked ? values.title : '',
+          author: values.isAuthorChecked ? values.author : '',
+          issn: values.isIssnChecked ? values.issn : '',
+          yearPublished: values.isYearPublishedChecked ? values.yearPublished : '',
+        });
+      }}>
+      {(formikProps: FormikProps<any>) => (
+        <Form>
+          <StyledFormWrapper>
+            <FormGroup>
+              <StyledFormElementWrapper>
+                <Field
+                  as={StyledFormControlLabel}
+                  type="checkbox"
+                  name="isDoiChecked"
+                  control={<Checkbox />}
+                  label={t('central_import.doi')}
+                  data-testid="duplicate-search-doi-checkbox"
+                  onClick={(event: any) => {
+                    if (event.target.value) {
+                      formikProps.setFieldValue('isTitleChecked', false);
+                      formikProps.setFieldValue('isAuthorChecked', false);
+                      formikProps.setFieldValue('isIssnChecked', false);
+                      formikProps.setFieldValue('isYearPublishedChecked', false);
+                    }
+                  }}
+                />
+                <Field name={'doi'}>
+                  {({ field }: FieldProps) => (
+                    <TextField
+                      data-testid="duplicate-search-doi-textfield"
+                      variant="outlined"
+                      {...field}
+                      fullWidth
+                      multiline
+                      disabled={!formikProps.values.isDoiChecked}
+                    />
+                  )}
+                </Field>
+              </StyledFormElementWrapper>
+
+              <StyledCenterWrapper>
+                <Typography>{t('central_import.or_search_with')}</Typography>
+              </StyledCenterWrapper>
+
+              <StyledFormElementWrapper>
+                <Field
+                  as={StyledFormControlLabel}
+                  type="checkbox"
+                  name="isTitleChecked"
+                  control={<Checkbox />}
+                  label={t('central_import.title')}
+                  onClick={(event: any) => {
```

Comment:
(hvis dette blir riktig type da)
samme under også
```suggestion
                  onClick={(event: ChangeEvent<HTMLInputElement>) => {
```

Reply:
stemmer,  avglemt. 
fikset